app/assets/stylesheets/v3/layout/variables.scss

```diff
@@ -58,6 +58,10 @@ $text-muted: #727272;
 $hyperlink-blue: $blue-main-brand;
 $dark-hyperlink-blue: $blue-main-brand--dark-med;
 
+/* Form Styles */
+$form-border-color: #A0A0A1;
+$form-border: 2px solid $form-border-color;
```

Comment:
I noticed from their invision files some forms use the light border and others a dark, how did you decide which one to use?

Reply:
I kind of went off of the comments in the Components phase 1 doc in teams, and then some of the invision files, but the styles seemed all over the map. so I just picked something 🤷 .